shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-api/src/main/java/org/apache/shardingsphere/readwritesplitting/api/rule/ReadwriteSplittingDataSourceRuleConfiguration.java

```diff
@@ -46,7 +62,16 @@
     public Optional<String> getAutoAwareDataSourceName() {
         return Optional.ofNullable(props.getProperty("auto-aware-data-source-name"));
     }
-    
+
+    @Test
+    public void assertGetAutoAwareDataSourceName(){
```

Comment:
This file does not need to be modified, you need to add a new test class like `ReadwriteSplittingRuleTest` and complete the assertion in the test class.